lib/buildEmbed.js

```diff
@@ -65,8 +83,7 @@ function defaultEmbed(id, options){
   out += options.noCookie ? "youtube-nocookie" : "youtube";
   out += '.com/embed/';
   out += id;
-  // autoplay is _technically_ possible, but be cool, don't do this
-  out += options.allowAutoplay ? '?autoplay=1' : '';
+  out += params ? "?" + params : "";
```

Comment:
Same thing with template literals here.

```suggestion
  out += params ? `?${params}` : "";
```